app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zero_if_age_less_than_first_year() -> None:
+    assert get_human_age(13, 13) == [0, 0]
```

Comment:
The test case checks for ages less than the first year, which is correct. However, ensure that you also test for edge cases such as negative numbers or extremely large values, as mentioned in the checklist.